test/runnable/imports/bar10378.d

```diff
@@ -0,0 +1,4 @@
+
+module bar;
```

Comment:
I know the feature, but this is not its case. This file is imported as the module name `imports.bar10378`, but the module declaration has a different fully qualified name `bar` which does not match.